software/sosse.yml

```diff
@@ -0,0 +1,13 @@
+name: "Sosse"
+website_url: "https://sosse.readthedocs.io/en/stable/"
+source_code_url: "https://gitlab.com/biolds1/sosse"
+description: "Selenium based search engine and crawler with offline archiving."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Python
+  - Docker
+tags:
+  - Search Engines
+  - Archiving and Digital Preservation (DP)
+depends_3rdparty: no
```

Comment:
```suggestion
```
By default, `depends_3rdparty` is set to "no", so there's no need to set it explicitly to "no" (Can be omitted).

Reply:
Done, thanks!